src/engines/html5/html5.js

```diff
@@ -190,9 +195,63 @@ export default class Html5 extends FakeEventTarget implements IEngine {
       this._eventManager.listen(this._mediaSourceAdapter, CustomEvents.TEXT_TRACK_CHANGED, (event: FakeEvent) => this.dispatchEvent(event));
       this._eventManager.listen(this._mediaSourceAdapter, CustomEvents.ABR_MODE_CHANGED, (event: FakeEvent) => this.dispatchEvent(event));
       this._eventManager.listen(this._mediaSourceAdapter, CustomEvents.TEXT_CUE_CHANGED, (event: FakeEvent) => this.dispatchEvent(event));
+      this._eventManager.listen(this._mediaSourceAdapter, CustomEvents.ERROR, (event: FakeEvent) => this.dispatchEvent(event));
+    }
+  }
+
+  /**
+   * Handles errors from the video element
+   * @returns {void}
+   * @private
+   */
+  _handleVideoError(): void {
+    if (!this._el.error) return;
+    const errCode = this._el.error.code;
+    if (errCode == 1 /* MEDIA_ERR_ABORTED */) {
+      // Ignore this error code.js, which should only occur when navigating away or
+      // deliberately stopping playback of HTTP content.
+      return;
     }
+
+    const error = new Error(Error.Severity.CRITICAL, Error.Category.MEDIA, Error.Code.VIDEO_ERROR, this._getErrorInfo());
+    this.dispatchEvent(new FakeEvent(CustomEvents.ERROR, error));
   }
 
+  /**
+   * more info about the error
+   * @returns {Object} info about the video element error
+   * @private
+   */
+  _getErrorInfo(): Object{
+    if (!this._el.error) return {};
+    let msInfo = this._el.error.msExtendedCode ? this._el.error.msExtendedCode : null;
+    // Extra error information from Chrome:
+    const chromeInfo = this._el.error.message ? this._el.error.message : null;
+    return {
+      errorCode: this._el.error.code,
+      chromeInfo: chromeInfo,
+      msInfo: this._getMSErrorInfo(msInfo)
```

Comment:
Looks very weird to send both `chromeInfo` and `msInfo`
call it just `info` and based on the runtime env pass the relevant info.